public/app/core/components/ForgottenPassword/ForgottenPassword.test.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { asyncScheduler, scheduled, throwError } from 'rxjs';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { ForgottenPassword, SendResetEmailDTO } from './ForgottenPassword';
+import { backendSrv } from '../../services/backend_srv';
+import { toAsyncOfResult } from '../../../features/query/state/DashboardQueryRunner/testHelpers';
+import { createFetchResponse } from '../../../../test/helpers/createFetchResponse';
+
+jest.mock('@grafana/runtime', () => ({
+  ...((jest.requireActual('@grafana/runtime') as unknown) as object),
+  getBackendSrv: () => backendSrv,
+}));
+
+function getTestContext() {
+  jest.clearAllMocks();
+  const fetchMock = jest.spyOn(backendSrv, 'fetch');
+  const { rerender } = render(<ForgottenPassword />);
+
+  return { fetchMock, rerender };
+}
+
+describe('ForgottenPassword', () => {
+  describe('when mounted', () => {
+    it('then it should show input field', () => {
+      getTestContext();
+
+      expect(screen.getByRole('textbox')).toBeInTheDocument();
```

Comment:
```suggestion
      expect(screen.getByRole('textbox', { name: /user/i })).toBeInTheDocument();
```
I think it's better to be more specific with the selector, otherwise if for some reason another input is added to the form, this test will break. For this to work, the `id` attribute will need to be added to the `Input`. This also promotes better accessibility as by adding that id to the Input, we'll make sure that the label is properly associated with it via the `for` attribute.

Reply:
Fixed!